activities/tool-management/plausible-analytics.md

```diff
@@ -4,20 +4,27 @@ type: resource
 
 # Plausible Analytics
 
-Plausible Analytics is a simple, lightweight (< 1 KB), open source and privacy-friendly analytics tool. It doesn’t use cookies. [See our Plausible dashboard](https://plausible.io/publiccode.net).
-
-We're currently trialling Plausible (see [issue 64](https://github.com/publiccodenet/jekyll-theme/issues/64)).
+Plausible Analytics is a simple, lightweight (< 1 KB), open source and privacy-friendly analytics tool.
+It doesn’t use cookies.
+[See our Plausible dashboard](https://plausible.io/publiccode.net).
 
 ## Using Plausible Analytics
 
-Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages). It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site.
+Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages).
+It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site.
 
 We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.
 
 ## Plausible management
 
-Plausible Analytics works by loading a Javascript script on the top navigation of our websites. [Our Plausible dashboard is publicly shared](https://plausible.io/publiccode.net). It can be configured by the codebase stewards and communication coordinator.
+Plausible Analytics works by loading a Javascript script on the top navigation of our websites.
+Our Plausible dashboards are publicly shared.
+You find them on the URL `https://plausible.io/[subdomain].publiccode.net`.
+For example: [plausible.io/blog.publiccode.net](https://plausible.io/blog.publiccode.net) or [plausible.io/publiccode.net](https://plausible.io/publiccode.net).
+It can be configured by the codebase stewards and communication coordinator.
 
 ## Privacy and GDPR
 
-Plausible doesn't collect or store any personal or identifiable information. Their data is hosted in Germany. Plausible is designed to be GDPR, CCPA and cookie law compliant. Read more in [Plausible's data policy](https://plausible.io/data-policy).
+Plausible doesn't collect or store any personal or identifiable information.
+Their data is hosted in Germany. Plausible is designed to be GDPR, CCPA and cookie law compliant.
```

Comment:
No linebreak between these two sentences?

Reply:
Missed that one, fixed now!